data/avro-schema/src/main/resources/avro/net/corda/data/membership/SignedContexts.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "SignedContexts",
+  "namespace": "net.corda.data.membership",
+  "doc": "Avro representation of the original byte data part which was signed in the MemberInfo.",
+  "fields": [
+    {
+      "name": "memberContext",
+      "doc": "Member provided data in MemberInfo serialised as byte array by using KeyValuePairList.",
+      "type": "bytes"
+    },
+    {
+      "name": "mgmContext",
+      "doc": "MGM provided data in MemberInfo serialised as byte array by using KeyValuePairList.",
```

Comment:
Where is the signature for this context?

Reply:
See my other comment for: https://github.com/corda/corda-api/pull/1131/files/4d64c47d4f14d69b292a8ea8b7705a0e9c76dac7#r1222728784